devicetypes/smartthings/zigbee-accessory-dimmer.src/zigbee-accessory-dimmer.groovy

```diff
@@ -23,6 +23,7 @@ metadata {
 		capability "Configuration"
 
 		fingerprint profileId: "0104", inClusters: "0000,1000,0003", outClusters: "0003,0004,0005,0006,0008,1000,0019", manufacturer: "Aurora", model: "Remote50AU", deviceJoinName: "Aurora Wireless Wall Remote"
+		fingerprint manufacturer: "IKEA of Sweden", model: "TRADFRI wireless dimmer", deviceJoinName: "IKEA TRÅDFRI Wireless dimmer"
```

Comment:
I'm realizing that although the fingerprints are a lot shorter, it was useful to see the cluster list. Thoughts on including it as a comment at the end of or above the fingerprint?

Reply:
if we're going to have the whole thing anyway we might as well just include it in the fingerprint.